crates/ruma-client-api/src/dehydrated_device/get_dehydrated_device.rs

```diff
@@ -0,0 +1,54 @@
+//! `GET /_matrix/client/*/dehydrated_device/`
+//!
+//! Get a dehydrated device for rehydration.
+
+pub mod unstable {
+    //! `msc3814` ([MSC])
+    //!
+    //! [MSC]: https://github.com/uhoreg/matrix-doc/blob/shrivelled_sessions/proposals/3814-dehydrated-devices-with-ssss.md
```

Comment:
Same about linking to PR

Reply:
Fixed as well in 89228a00494157aa6dc438c293ef0be7cf316c51.